web/sites/default/config/search_api.index.full_index.yml

```diff
@@ -889,4 +889,4 @@ options:
   delete_on_fail: true
   index_directly: true
   track_changes_in_references: false
-server: pantheon_solr8
+server: pantheon_search
```

Comment:
Just a cautionary tale: when I made this change in PGOV. The Live site lost all search index after deployment until the full re-indexing process is done.